Add visualization (#113)
aramis-lab/AD-DL: docs/RandomSearch.md

Optional arguments:

- **Computational resources**
    - `--use_cpu` (bool) forces to use CPU. Default behaviour is to try to use a GPU and to raise an error if it is not found.
    - `--nproc` (int) is the number of workers used by the DataLoader. Default value: `2`.
    - `--batch_size` (int) is the size of the batch used in the DataLoader. Default value: `2`.
    - `--evaluation_steps` (int) gives the number of iterations to perform an [evaluation internal to an epoch](Details.md#evaluation). 
    Default will only perform an evaluation at the end of each epoch.
- **Cross-validation arguments**
    - `--n_splits` (int) is a number of splits k to load in the case of a k-fold cross-validation. Default will load a single-split.
    - `--split` (list of int) is a subset of folds that will be used for training. By default all splits available are used. 
    
## Content of `random_search.json`

`random_search.json` must be present in `launch_dir` before running the command. 
An example of this file can be found in 
[clinicadl/clinicadl/tools/deep_learning/models](https://github.com/aramis-lab/AD-DL/tree/master/clinicadl/clinicadl/tools/deep_learning/models).

Mandatory variables:

> - `mode` (str) is the type of input used. Must be chosen between `image`, `patch`, `roi` and `slice`. 

Reviewer: ```suggestion
- `mode` (str) is the type of input used. Options are: `image`, `patch`, `roi` and `slice`. 
```
